src/Security/Security/help/New-AzSqlVulnerabilityAssessmentBaseline.md

```diff
@@ -0,0 +1,301 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
+Module Name: Az.Security
+online version:
```

Comment:
Please add online version. BTW, please rename `New-AzSqlVulnerabilityAssemetBaseline.md` to `Add-AzSqlVulnerabilityAssemetBaseline.md` according to design review.

Reply:
pending